Sources/Apodini/Semantic Model Builder/OpenAPI/OpenAPIInterfaceExporter.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Created by Paul Schmiedmayer on 11/3/20.
+//
+
+@_implementationOnly import OpenAPIKit
+@_implementationOnly import Vapor
+import Foundation
+
+class OpenAPIInterfaceExporter: InterfaceExporter {
+    let app: Application
+    var documentBuilder: OpenAPIDocumentBuilder
+    let configuration: OpenAPIConfiguration
+
+    required init(_ app: Application) {
+        self.app = app
+        self.configuration = OpenAPIConfiguration.create(from: app)
+        self.documentBuilder = OpenAPIDocumentBuilder(
+                configuration: configuration
+        )
+    }
+
+    func export<H: Handler>(_ endpoint: Endpoint<H>) {
+        documentBuilder.addEndpoint(endpoint)
+    }
+
+    func finishedExporting(_ webService: WebServiceModel) {
+        serveSpecification()
+    }
+
+    private func serveSpecification() {
+        // swiftlint:disable:next todo
```

Comment:
Again, I'd rather not have TODOs in code. Why not create an issue 🙂

Reply:
#107